### ISSUE TITLE
moving to vuejs 3

### ISSUE THREAD

**Issue by petrus-v** [AnyBlok/anyblok_furetui#56]
As far I understand, Buefy do not plan to move to vue 3 as suggested here https://github.com/buefy/buefy/issues/2505#issuecomment-997000720 it would needs to move on [Oruga](https://oruga.io/).

Do you have any plan ?

**Comment by jssuzanne** [comment]
Yes, I tried last year, but it was to early. I need time and help :D

**Comment by jssuzanne** [comment]
I thought last night.

There are other libs in the same case. You have to act in 2 phases

1) use v2 / v3 compatible libs
2) upgrade to v3
